### PR TITLE
mo-static merge

### PR DESCRIPTION
**Pull request by joesolly**

### CODE REVIEW

**Comment by recursivefunk** [comment]
👍

**Comment by joesolly** [comment]
@jcarbaugh what do you think?

**Comment by akrawchyk** [comment]
@joesolly can you remove the mo-static part? this PR contains 2 discrete sets of changess, i opened https://github.com/istrategylabs/mo-django/pull/50 with regards to mo-static.

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings.py [hunk 1]
```diff
@@ -113,6 +113,23 @@
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 SSLIFY_DISABLE = config('SSLIFY_DISABLE', default=False, cast=bool)
 
+{% if cookiecutter.use_redis == "y" -%}
+
+# redis
+
+SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
+CACHES = {
+    "default": {
```
Comment (akrawchyk) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings.py]: Please change double quotes to single quotes to maintain consistency with the rest of the file.

**Comment by joesolly** [comment]
@akrawchyk this PR is finished, why don't we just pull this one instead of #50?

**Comment by jcarbaugh** [comment]
👍 for the code as-is, pending @joesolly and @akrawchyk deciding on the scope of this ticket vs #50

**Comment by akrawchyk** [comment]
@jcarbaugh we paired on this and closed #50 in favor of merging this PR. 👍

**Pull request merged**